### DISCUSSION TITLE
Display the content of internal linked page.

### DISCUSSION THREAD

**Discussion by dkkb** [outline/outline#2792] [Ideas]
Related to https://github.com/outline/outline/issues/1352.

I discovered that hovering over an internal link displays the 'HoverPreview' page, which I think is a great feature. It's also used by Github to display issues that are connected.

<img src="https://user-images.githubusercontent.com/82504881/142794551-5843bb2f-5cb1-4278-80ab-fecd0da743bd.png" width=400>

We can do better, in my opinion. What about adding the ability to directly display the content of the internal link page?

Many other notes software supports 'block,' which means we can embed internal pages as a block and have them display immediately like a 'iframe.'

If the users don't use this feature, just display the internal link by default. We can add a query to the link if the users choose this option, like `/doc/learning-OjnWSaEpYK?OUTLINE_SHOW=true`, and the content of this connected page will be displayed instantly in outline.

**Comment by tommoor** [comment]
Do you have an example of this in other software?

**Reply by dkkb** [reply]
https://help.obsidian.md/How+to/Link+to+blocks
Obsidian supports `Link to blocks` and `Embed or transclude blocks`

**Comment by CaneTLOTW** [comment]
Would even like to see this for external links.
Like what we already have for Google documents, Youtube and other stuff.

**Comment by joezappie** [comment]
Being able to embed another internal page would be awesome!

An example case I have right now is I've put our company handbook in a document and there are some things in that handbook that would be good to have as their own page for quick access such as our vacation policy.

Being able to make a "vacation" doc and then embed that into our handbook page would keep things in sync instead of managing the info in multiple spots.

I get that people could also search for "vacation" and the handbook would come up, but it doesn't auto scroll to the vacation section and vacation is mentioned in multiple parts of a very long handbook. Being able to break important but useful info into separate pages but have it still in the main handbook is the use case.

I could use this feature in tons of my pages though, from documenting software to IT user guides.
